cli/main.go

```diff
@@ -16,11 +16,18 @@ import (
 	"github.com/spf13/afero"
 )
 
+const (
+	default_version = "dev"
+	default_commit  = "none"
+	default_date    = "unknown"
+	default_builder = "unknown"
```

Comment:
No snake_case comstants in Go

I'm sure there is a revive linter about it 😅

Maybe it could be enabled